src/main/java/org/broadinstitute/hellbender/tools/spark/sv/CollectInsertSizeMetricsSpark.java

```diff
@@ -196,11 +194,12 @@ public ReadFilter makeReadFilter() {
                                            .and(ReadFilterLibrary.READLENGTH_EQUALS_CIGARLENGTH)
                                            .and(ReadFilterLibrary.SEQ_IS_STORED)
                                            .and(ReadFilterLibrary.CIGAR_IS_SUPPORTED)
-                                           .and(ReadFilterLibrary.MAPPING_QUALITY_AVAILABLE);
+                                           .and(ReadFilterLibrary.MAPPING_QUALITY_AVAILABLE)
+                                           .and(ReadFilterLibrary.MAPPED);
 
         @Override
         public boolean test(final GATKRead read){
-            return primary.test(read);
+            return primary.test(read) && (!read.mateIsUnmapped());
```

Comment:
You're already including ReadFilterLibrary.MAPPED in this so I don't think you need to test that again here.